pkg/issuer/acme/dns/dns.go

```diff
@@ -54,57 +54,28 @@ func (s *Solver) Present(ctx context.Context, crt *v1alpha1.Certificate, domain,
 	if err != nil {
 		return err
 	}
-	glog.V(4).Infof("Presenting DNS01 challenge for domain %q", domain)
+	glog.Infof("Presenting DNS01 challenge for domain %q", domain)
```

Comment:
Why change this log level?

Reply:
Given the chart doesn't deploy with --v=4 by default, and given this information is useful to basically anyone debugging acme issues with cert-manager (i.e. the majority of users), it seemed sensible to increase the log level to make it easier for users to debug out of the box. Let's see if we have complaints about excessive log messages after merging this 😄